assets/js/index.js

```diff
@@ -93,10 +93,8 @@ twentytwenty.coverModals = {
 	// Handle cover modals when they're toggled
 	onToggle: function() {
 		document.querySelector( '.cover-modal' ).addEventListener( 'toggled', function( event ) {
-			var modal, body;
-
-			modal = event.target;
-			body = document.body;
+			var modal = event.target,
+				body = document.body;
```

Comment:
The indentation looks off in this situations :)

Reply:
Also, sorry for coding them like this, that's how the previous linter wanted them...

Reply:
The indentation is required by the handbook. https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#declaring-variables-with-var